src/shared/bus-util.c

```diff
@@ -769,6 +770,23 @@ int bus_print_property(const char *name, sd_bus_message *property, bool value, b
                         char timespan[FORMAT_TIMESPAN_MAX];
 
                         print_property(name, "%s", format_timespan(timespan, sizeof(timespan), u, 0));
+                } else if (streq(name, "RestrictNamespaces")) {
+                        _cleanup_free_ char *s = NULL;
+                        const char *result = NULL;
+
+                        if (!u)
```

Comment:
hmm, see CODING_STYLE: unless you check for NULL or check a boolean please do not rely on C's downgrade-to-bool feature. Specifically, if you compare with numbers, please write `== 0` explicitly.

That said, on the PID 1 side we so far explicitly masked out non-relevant bits from the namespace mask here before comparing, and I think we should do the same here... i.e. `if (u & NAMESPACE_FLAGS_ALL == 0) …`